fastlane/lib/fastlane/environment_printer.rb

```diff
@@ -146,7 +146,7 @@ def self.print_system_locale
       env_table = ""
       ["LANG", "LC_ALL", "LANGUAGE"].each do |e|
         env_icon = "🚫"
-        if ENV[e] && ENV[e].end_with?("UTF-8")
+        if FastlaneCore::Env.enabled?(e) && ENV[e].end_with?("UTF-8")
```

Comment:
I think the intent of this line was to check if `ENV[e]` was `nil`, not to see if it was enabled.